code/datums/jobs.dm

```diff
@@ -2189,8 +2208,12 @@
 /datum/job/special/meatcube
 	name = "Meatcube"
 	linkcolor = "#FF0000"
-	limit = 0
+	limit = 0 + (ASS_JAM * 300)
+#if ASS_JAM
```

Comment:
Again, the change to limit could have easily been in the #if/#else/#endif. Additionally, does this make meatcube valid for traitor? This is perhaps not wise.

Reply:
Meatcube is indeed valid for traitor, but its latejoin only, so you sort of know what you're getting into.